Shop-G/ShopGo
# build menu button

##jira
TS-22
Description:
• implemented functionality of hamburger menu button for homeViewController

How to test it?
• navigate to homeViewController and press the hamburger menu button on top left corner.
• menu will appear by pressing the hamburger menu button.
• menu will disappear by pressing the same hamburger menu button
